### PR TITLE
fix: remove namespace from error message

### LINKED ISSUE #10392: Namespace field in New Application panel is confusing
**Describe the bug**

The new `Application Namespace` field is confusing. Users ( including myself ) confuse this field with the project field. The error message sounds like the application destination namespace is not allowed in the project.

**To Reproduce**

Reproduced it trying to create guestbook app in new Argo CD version

**Expected behavior**

I think `Application Namespace` field should be hidden unless multiple namespace feature is enabled.

**Screenshots**

Screen Shot 2022-08-18 at 1 23 27 PM

**Version**

```shell
master (v2.5.0-unreleased)
```

### PR DESCRIPTION
**Pull request by ashutosh16**
Signed-off-by: Ashutosh <mail.ashutosh8@gmail.com>

fix  [argo-cd/issues/10392](https://github.com/argoproj/argo-cd/issues/10392)
Note on DCO:

If the DCO action in the integration test fails, one or more of your commits are not signed off. Please click on the *Details* link next to the DCO action for instructions on how to resolve this.

Checklist:

* [ ] Either (a) I've created an [enhancement proposal](https://github.com/argoproj/argo-cd/issues/new/choose) and discussed it with the community, (b) this is a bug fix, or (c) this does not need to be in the release notes.
* [ ] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] I've included "Closes [ISSUE #]" or "Fixes [ISSUE #]" in the description to automatically close the associated issue.
* [ ] I've updated both the CLI and UI to expose my feature, or I plan to submit a second PR with them.
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [ ] Optional. My organization is added to USERS.md.
* [ ] I have signed off all my commits as required by [DCO](https://github.com/argoproj/argoproj/tree/master/community#contributing-to-argo)
* [ ] I have written unit and/or e2e tests for my change. PRs without these are unlikely to be merged.
* [ ] My build is green ([troubleshooting builds](https://argo-cd.readthedocs.io/en/latest/developer-guide/ci/)).

### CODE REVIEW

File: util/argo/argo.go [hunk 1]
```diff
@@ -422,8 +422,8 @@ func GetAppProject(app *argoappv1.Application, projLister applicationsv1.AppProj
 		return nil, err
 	}
 	if !proj.IsAppNamespacePermitted(app, ns) {
-		return nil, fmt.Errorf("application '%s' in namespace '%s' is not allowed to use project '%s'",
-			app.Name, app.Namespace, proj.Name)
+		return nil, fmt.Errorf("application '%s' is not allowed to use project '%s'",
```
Comment (ashutosh16) [inline util/argo/argo.go:425]: Remove the Application namespace option from the UI 
Screen Shot 2022-08-29 at 6 34 56 PM

File: util/argo/argo.go [hunk 2]
```diff
@@ -422,8 +422,8 @@ func GetAppProject(app *argoappv1.Application, projLister applicationsv1.AppProj
...
+			app.Name, proj.Name)
```
Comment (jannfis) [inline util/argo/argo.go:426]:
```suggestion
		return nil, fmt.Errorf("application '%s' is not allowed to use project '%s'",
			app.QualifiedName(), proj.Name)
```

Comment (jannfis) [reply util/argo/argo.go:426]: IMHO, the namespace is integral part of the error message.

Using `app.QualifiedName()` will result in a `<namespace>/<name>` string, so the error message would be:

`"application 'foo/bar' is not allowed to use project 'baz'"`

Comment (ashutosh16) [reply util/argo/argo.go:426]: make sense.

Comment (jannfis) [reply util/argo/argo.go:426]: Hm, seems you haven't pulled that change in.

**Comment by ashutosh16** [comment]
@jannfis  I closed this PR, as my rebase went wrong. I don't want to open another PR for the small change. Would you be able to add the change you suggested
https://github.com/ashutosh16/argo-cd/blob/2aaadcabfc4da4a778e7182cede15b11d630dfca/util/argo/argo.go#L425

**Pull request closed without merging**